buildVars.py

```diff
@@ -17,7 +17,7 @@
 	"addon_summary" : _("Report Symbols"),
 	# Add-on description
 	# Translators: Long description to be shown for this add-on on add-on information from add-ons manager
-	"addon_description" : _("""Allows to listen the typed symbols (non alphanumeric characters), even when the speaking of characters is turned off."""),
+	"addon_description" : _("""Allows to listen the typed symbols (non alphanumeric characters or blank characters), even when the speaking of characters is turned off."""),
```

Comment:
Thanks for your work. I think we can omit a reiteration of the word characters. I will finish this.